downstream/modules/platform/proc-controller-create-insights-credential.adoc

```diff
@@ -2,7 +2,28 @@
 
 = Creating Red Hat Insights credentials
 
-Use the following procedure to create a new credential for use with Red Hat Insights:
+.Prerequisites
+
+* To use token-based authentication, you must link:https://docs.redhat.com/en/documentation/red_hat_hybrid_cloud_console/1-latest/html/creating_and_managing_service_accounts/proc-ciam-svc-acct-overview-creating-service-acct#proc-ciam-svc-acct-create-creating-service-acct[create a Red Hat service account] to generate a *Client ID* and *Client secret*. 
+* Assign this service account to the appropriate *User Access* group with necessary permissions. 
+
+To enable integration between {PlatformNameShort} and {InsightsShort}, assign the service account with the following permissions:
+
+* *inventory:hosts:read* (included in the Inventory Hosts viewer role)
+* *patch::read* (included in the Patch viewer role)
```

Comment:
patch isn't necessary, so I'd leave this out

it might be required for someone to create a remediation through console.redhat.com, but AAP itself doesn't require it